components/expression_language.rst

```diff
@@ -121,8 +125,8 @@ This is how you can use these flags::
 
     $expressionLanguage = new ExpressionLanguage();
 
-    // this returns true because the unknown variables and functions are ignored
-    var_dump($expressionLanguage->lint('unknown_var + unknown_function()', Parser::IGNORE_UNKNOWN_VARIABLES | Parser::IGNORE_UNKNOWN_FUNCTIONS));
+    // this returns null because the unknown variables and functions are ignored, but don't throw `SyntaxError` exception
```

Comment:
It will return nothing if there is no error

Reply:
Is that not what is expected ?

Reply:
yes, but the comment talks about "it returns null" which is not the same :)

Reply:
maybe rewrite the comment like this:

```php
// does not throw a SyntaxError because the unknown variables and functions are ignored
```

Reply:
Ok my bad, I did not understand it correctlly.

I'll keep your example. thx